### ISSUE TITLE
Json outupt - SyntaxError: JSON.parse

### ISSUE THREAD

**Issue by dawidmalina** [keilerkonzept/terraform-module-versions#18]
Json outupt cannot be parsed and consumed by other tools as json is not valid:
```
{
  "path": "examples/main.tf",
  "name": "consul",
  "constraint": "> 0.1.0",
  "constraintUpdate": true,
  "latestMatching": "0.7.7",
  "latestOverall": "0.7.7"
}
{
  "path": "examples/0.12.x.tf",
  "name": "consul_aws",
  "constraint": ">=0.5.0,<=1.0.0",
  "constraintUpdate": true,
  "latestMatching": "0.7.7",
  "latestOverall": "0.7.7"
}
{
  "path": "examples/main.tf",
  "name": "consul_github_https",
  "constraint": "0.7.3",
  "latestMatching": "0.7.3",
  "latestOverall": "0.7.7",
  "nonMatchingUpdate": true
}
```

it should looks like this:
```
[{
  "path": "examples/main.tf",
  "name": "consul",
  "constraint": "> 0.1.0",
  "constraintUpdate": true,
  "latestMatching": "0.7.7",
  "latestOverall": "0.7.7"
},
{
  "path": "examples/0.12.x.tf",
  "name": "consul_aws",
  "constraint": ">=0.5.0,<=1.0.0",
  "constraintUpdate": true,
  "latestMatching": "0.7.7",
  "latestOverall": "0.7.7"
},
{
  "path": "examples/main.tf",
  "name": "consul_github_https",
  "constraint": "0.7.3",
  "latestMatching": "0.7.3",
  "latestOverall": "0.7.7",
  "nonMatchingUpdate": true
}]
```

**Comment by sgreben** [comment]
it's [JSONLines](http://jsonlines.org/) :) Often more convenient than arrays.

```sh
$ terraform-module-versions -update examples | jq 'select(.matchingUpdate)'
{
  "path": "examples/main.tf",
  "name": "example_git_ssh",
  "constraint": "~> 0.10",
  "version": "0.10.0",
  "constraintUpdate": true,
  "latestMatching": "0.11.2",
  "matchingUpdate": true,
  "latestOverall": "2.0.8",
  "nonMatchingUpdate": true
}
```

You can collect the objects into an array using `jq -s`:

```sh
$ terraform-module-versions -update examples | jq 'select(.matchingUpdate)' | jq -s
[
  {
    "path": "examples/main.tf",
    "name": "example_git_ssh",
    "constraint": "~> 0.10",
    "version": "0.10.0",
    "constraintUpdate": true,
    "latestMatching": "0.11.2",
    "matchingUpdate": true,
    "latestOverall": "2.0.8",
    "nonMatchingUpdate": true
  }
]
```

I think we should have both of -o json, -o jsonl when we add the formatting options.

**Comment by sgreben** [comment]
addressed in #25 :)
